### PR TITLE
adds video captioning section

### PR DESCRIPTION
**Pull request by jaredcunha**
Address issue #33 

This adds a section for video captioning and points to reference docs. I have couple questions:
- Should explain open vs. closed captioning?
- Is "Video captioning" the best title if we don't want to specify open vs. closed?
- Should we address transcripts?
- Reference contrast levels in the steps?

Let me know what you think.

### CODE REVIEW

**Comment by maya** [comment]
Thanks so much for doing this @jaredcunha! Will check with the team and get back to you.

File: pages/content/closed-captioning.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
+layout: primary
+title: Video captioning
```
Comment (davidstenger) [inline pages/content/closed-captioning.md]: can we change this to: "Video and multimedia"

File: pages/content/closed-captioning.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+permalink: /content-design/video-captioning/
+category: Content design
+sidebar: content
+question: Is everyone able to access your multimedia content?
+personas:
+- Blake is hearing-impaired and cannot rely on audio.
+- Sandra is a non-native English speaker and has difficulty understanding video.
+---
+
+### Steps to take
```
Comment (davidstenger) [inline pages/content/closed-captioning.md]: We don't talk about making sure we have a transcript.

File: pages/content/closed-captioning.md (hunk 1 above)
Comment (jaredcunha) [reply pages/content/closed-captioning.md]: Sure. Should probably reflect that in the URLs as well, right?

File: pages/content/closed-captioning.md (hunk 2 above)
Comment (jaredcunha) [reply pages/content/closed-captioning.md]: Were you saying that I haven't mentioned a case in which someone may need a transcript? Or were you pointing out the case of Sandra would not be applicable here? I wasn't too clear.

Comment (maya) [reply pages/content/closed-captioning.md]: Since the comment was on "Steps to take," my guess is he meant the former.

**Comment by jeremyzilar** [comment]
This would be a timely and useful addition. We are trying to improve our video captioning across all of digital.gov, and would love to link this as we talk about our process. 

I also think there are people in the Video community of practice who would like to weigh in once this is up and live https://digital.gov/communities/video-production/ Happy to help connect them with this page.

File: pages/content/closed-captioning.md (hunk 1 above)
Comment (maya) [reply pages/content/closed-captioning.md]: Done in 5284cce and 9da2627.

**Comment by maya** [comment]
Going to merge this into the demo branch so that the team can see how it looks. We can also do the edits in another PR to the demo branch.

**Comment by maya** [comment]
Here's a preview of what it looks like: https://demo.accessibility.digital.gov/content-design/video-and-multimedia/

**Pull request merged**